webapp/src/components/confirm_feedback_modal/confirm_feedback_modal.jsx

```diff
@@ -37,7 +50,18 @@ export default class ConfirmFeedbackModal extends React.PureComponent {
                     <Modal.Title>{'Send Feedback'}</Modal.Title>
                 </Modal.Header>
                 <Modal.Body>
-                    {'You are about to send feedback about Mattermost. Are you sure?'}
+                    <p>{'You are about to send feedback about Mattermost. Are you sure?'}</p>
```

Comment:
```suggestion
                    <p>{'Send us your feedback and suggestions.'}</p>
```

Reply:
@justinegeffen at this stage the user already typed his feedback and sent the message. We are trying to confirm that they are indeed sure they want to send it

Reply:
Ah! Gotcha. Ok, so then let's go with:

"You're about to send feedback about Mattermost."

Reply:
Without the "Are you sure?"